db/migrate/20230115233416_create_sidebar_sections.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class CreateSidebarSections < ActiveRecord::Migration[7.0]
+  def change
+    create_table :sidebar_sections do |t|
+      t.integer :user_id, null: false
+      t.string :title, null: false
+      t.timestamps
+    end
+
+    add_index :sidebar_sections, %i[user_id title], unique: true
```

Comment:
Actually do we need a unique index on `user_id` and `title` right now when a user can only have one custom section? I think we should only have a unique index on `user_id`. Otherwise, a user can create multiple section as long as the sections have different titles.

Reply:
Right now UI is allowing you to create multiple sections

<img width="245" alt="Screenshot 2023-01-31 at 3 17 27 pm" src="https://user-images.githubusercontent.com/72780/215661808-5813a76d-b7c3-47b8-95d3-463e9a607d60.png">

Reply:
OK just wanted to confirm from product discussion that we're going with multiple custom sections because my last impression is that we are only going to support one.